src/lib.rs

```diff
@@ -1,37 +1,304 @@
-use lazy_static::lazy_static;
-use std::process;
+//! This crate aims to provide an easy and customizable way to help you build
+//! Wasm projects by extending them with custom subcommands, based on the
+//! [`xtask` concept](https://github.com/matklad/cargo-xtask/), instead of using
+//!  external tooling like [`wasm-pack`](https://github.com/rustwasm/wasm-pack).
+//!
+//! # Minimum Supported Rust Version
+//!
+//! This crate requires **Rust 1.58.1** at a minimum.
```

Comment:
```suggestion
//! This crate requires **Rust 1.58.1** at a minimum because there is a security issue on a function we use in std in previous versions.
```